Xamarin.Forms.Platform.Android/Renderers/EditorRenderer.cs

```diff
@@ -177,5 +182,23 @@ void OnKeyboardBackPressed(object sender, EventArgs eventArgs)
 			ElementController?.SendCompleted();
 			Control?.ClearFocus();
 		}
+
+        void UpdateMaxLength()
+        {
+            var currentFilters = new List<IInputFilter>(Control?.GetFilters() ?? new IInputFilter[0]);
+
+            for (var i = 0; i < currentFilters.Count; i++)
```

Comment:
Could be done more concisely with LINQ.?
Ie, `var currentFilters = currentFilters.Where(x=>!(x is InputFilterLengthFilter));`

Reply:
LINQ isn't recommended, as it isn't very performant. I think the XF team will reject anything with LINQ in it. They have implied that to me in the past, and you see it in their comments around the place in the code too, that they don't want it. for loops are the best, foreach if you need to use the variable in the loop more than once.

Reply:
Given these are renderers and can't be unit tested, I would have thought the risk of bugs would far outweigh the minimal perf difference of linq on small in memory data structures. I would be very surprised if performance testing would have revealed linq queries to be the culprit of any real life perf issues in XF. Having said that, this is 100% their perogative, so it's good to know so we can adhere to it